### PR TITLE
Added new instruction in startscreen

### PR DESCRIPTION
**Pull request by Monal5031**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -4,6 +4,8 @@
         1. You are supposed to make as many words as you can from the given set of letters in the grid.
         \n2. If the submitted word is a valid one then it is added to the list and the score would be updated.
         \n3. The minimum valid word length is 4 i.e. "cars" is a valid word whereas "car" is not.
+        \n4. You can select letters adjacent to currently selected letter's only.
```
Comment (aviaryan) [inline app/src/main/res/values/strings.xml]: Make this the 2nd point and `letter's` should be `letters`.  Also adjacent is not a correct term as it doesn't represent squares sharing a vertex.

**Review by aviaryan — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -4,6 +4,8 @@
         1. You are supposed to make as many words as you can from the given set of letters in the grid.
         \n2. If the submitted word is a valid one then it is added to the list and the score would be updated.
         \n3. The minimum valid word length is 4 i.e. "cars" is a valid word whereas "car" is not.
+        \n4. You can select letters joining currently selected letters only.
```
Comment (aviaryan) [inline app/src/main/res/values/strings.xml]: Please make this the 2nd point as it is best suited after 1st point.

**Review by aviaryan — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (Monal5031) [reply app/src/main/res/values/strings.xml]: Yes will do it, also the `letters` should be `letter`. Reason is I tested the app again we can only select the letter which is next to only the previously selected letter not all of them.

Comment (aviaryan) [reply app/src/main/res/values/strings.xml]: ...you can select letters such that consecutive letters are neighbour to one another

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -2,8 +2,10 @@
     <string name="app_name">Hinix</string>
     <string name="instructions">
         1. You are supposed to make as many words as you can from the given set of letters in the grid.
-        \n2. If the submitted word is a valid one then it is added to the list and the score would be updated.
+        \n2. You can onyl select letters such that consecutive letters are neighbour to one another.
```
Comment (aviaryan) [inline app/src/main/res/values/strings.xml]: onyl

**Review by aviaryan — CHANGES_REQUESTED** [review]
😆

**Review by aviaryan — APPROVED** [review]
✅

**Pull request merged**